app/game-master/reducers/score.js

```diff
@@ -1,80 +1,47 @@
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc6', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc7', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc8', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc9', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc10', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc11', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc12', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc13', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc14', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc15', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc16', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc17', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc18', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc19', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc20', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc21', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc22', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc23', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc24', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc25', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc26', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc27', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc28', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc29', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc30', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc31', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc32', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc33', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc34', time: 25 },
-        { displayName: 'John Wick', participantId: 'qwe-asd-zxc35', time: 25 },
-    ],
+import MockData from '../mock-data/score';
+import {
+    PARTICIPANT_JOINED,
+    PARTICIPANT_LEFT,
+} from '../actions/participant';
+import { PARTICIPANT_SOLUTION } from '../actions/round';
+
+const defaultState = MockData;
+const defaultParticipant = {
+    time: null,
+    length: 0,
+    correct: false,
 };
 
 export default function participant(state = defaultState, action) {
-    switch(action.type) {
+    switch (action.type) {
+        case PARTICIPANT_JOINED: {
+            const newParticipant = Object.assign(defaultParticipant, action.payload);
```

Comment:
`Object.assign` will modify the "origin". Better to clone it with `Object.assign({}, defaultParticipant, action.payload);`. What do you think?